core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -245,6 +245,80 @@ class Constants {
    */
   readonly keys_flags_extend = 1;
 
+  /**
+   * Constant for no modifiers
+   */
+  readonly keys_mod_none = 0;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_altL = 1 << 2;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_altR = 1 << 3;
+  /**
+   * bitmask for either alt
```

Comment:
```suggestion
   * bitmask for either Alt (Windows) or Option (Apple) modifier key
```

Do we need to have a code for a chirality-agnostic version of this, independently? Haven't thought through all the possibilities here yet.

Reply:
trying to match the CLDR spec where `alt` can be a keyword meaning both.